Sources/OpenAI/AIProxy/Endpoint+AIProxy.swift

```diff
@@ -124,5 +126,9 @@ private func getDeviceCheckToken() async -> String? {
 
 /// Get a unique ID for this user (scoped to the current vendor, and not personally identifiable):
 private func getVendorID() -> String? {
+#if canImport(UIKit)
     return UIDevice.current.identifierForVendor?.uuidString
```

Comment:
For my learning, what is the whole purpose of the identifierForVendor?

Reply:
This is not used for anything right now on the backend, although I hope to use it soon. Part of the vision for AIProxy is to provide a developer dashboard where they can see an AI usage breakdown of their customers  (e.g. what the distribution of requests looks like). So, for example, a view in the dashboard could have 'requests sent by user', and the `identifierForVendor` would be used to bucket the requests. See this mockup, the identifier would be used in the 'user ID' column:
![dashboard](https://github.com/jamesrochabrun/SwiftOpenAI/assets/35940/0771b61a-f528-4f2c-bc1c-9078666db3b7)

Reply:
I see, nice yes a breakdown is key here!

Reply:
Also follow-up question, how would this breakdown work if the developer has a Mac app?

Reply:
I was surprised to find last night that `identifierForVendor` is not part of the AppKit API. I have yet to do research on an alternative, but I'm sure there is one. I can test out a couple options